Apigee/ManagementAPI/DeveloperAppAnalytics.php

```diff
@@ -155,17 +157,18 @@ public function getAllEnvironments()
      *
      * @return array
      */
-    public function getByAppName($devIdOrCompanyName, $appName, $metric, $tStart, $tEnd, $tUnit, $sortBy, $sortOrder = 'ASC')
+    public function getByAppName($devEmailOrCompany, $is_company, $appName, $metric, $tStart, $tEnd, $tUnit, $sortBy, $sortOrder = 'ASC')
```

Comment:
I do not like to idea of introducing a new mandatory parameter not in the end of the parameter list. This could cause a breaking change in consumer apps.